.github/stale.yml

```diff
@@ -0,0 +1,58 @@
+# Configuration for probot-stale - https://github.com/probot/stale
```

Comment:
Hmmm do we want these changes as well here? I don't think they have an effect on this repo I guess.

Reply:
If a change isn't affecting downstream repo, then I don't see any reason to diverge git history, which in the long term might cause merge conflicts.